framework/apis/config/v1alpha1/config.proto

```diff
@@ -30,6 +30,8 @@ message Limiter {
   }
   RateLimitBackend backend = 3;
   google.protobuf.Duration refresh = 4 [(gogoproto.stdduration) = true];
+  bool DisableAdaptiveRateLimit = 5;
```

Comment:
camel style like "disableAdaptiveRateLimit".

change var name `Multicluster` as well, as it's not been used until now.